lte/gateway/c/core/oai/test/amf/test_amf_stateless.cpp

```diff
@@ -1034,13 +1024,8 @@ TEST_F(AMFAppStatelessTest, TestAfterPDUSessionReleaseComplete) {
 
   // Calling pseudo_amf_restart() and SetUp() simulates a service restart.
   AMFAppStatelessTest::pseudo_amf_restart();
-
-  EXPECT_TRUE(amf_state_ue_id_ht->isEmpty());
-  EXPECT_TRUE(
-      amf_app_desc_p->amf_ue_contexts.gnb_ue_ngap_id_ue_context_htbl.isEmpty());
-  EXPECT_TRUE(amf_app_desc_p->amf_ue_contexts.imsi_amf_ue_id_htbl.isEmpty());
-  EXPECT_TRUE(amf_app_desc_p->amf_ue_contexts.tun11_ue_context_htbl.isEmpty());
-  EXPECT_TRUE(amf_app_desc_p->amf_ue_contexts.guti_ue_context_htbl.isEmpty());
+  EXPECT_TRUE(AmfNasStateManager::getInstance().get_ue_state_map()->isEmpty());
+  EXPECT_EQ(get_amf_nas_state(false), nullptr);
```

Comment:
@shashidhar-patil , is this to confirm that no state exists in the Redis DB after restart? Could you please add a comment here to explain this?

Reply:
This checks if the state data in Amf is cleared after pseudo_amf_restart(). Added this comment.

 Also renamed `pseudo_amf_restart()` to `pseudo_amf_stop()` , as this is a more appropriate name. Calling `pseudo_amf_stop()` and `SetUp()` simulates amf service restart.

Reply:
Thanks for the cleanup!